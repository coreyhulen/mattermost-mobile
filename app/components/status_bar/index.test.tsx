// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import {Preferences} from '@constants';
import React from 'react';

import {renderWithIntl} from '@test/intl-test-helper';

import StatusBar from './index';

describe('@components/app_version', () => {
    it('should match snapshot', () => {
        const wrapper = renderWithIntl(<StatusBar theme={Preferences.THEMES.default}/>);
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});
