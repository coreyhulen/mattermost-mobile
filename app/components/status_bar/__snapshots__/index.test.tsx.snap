// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@components/app_version should match snapshot 1`] = `null`;
